<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSettings</class>
 <widget class="QDialog" name="DialogSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>159</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="4">
    <widget class="QCheckBox" name="checkAutoRun">
     <property name="text">
      <string>Автоматически запускать приложение при запуске системы</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QCheckBox" name="checkTray">
     <property name="text">
      <string>Сворачивать приложение в трей при закрытии окна</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QKeySequenceEdit" name="keySequenceEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Захват пикселя</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>+ ЛКМ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Поддерживаются модификаторы Ctrl, Shift, Alt</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
